certified-connectors/Sociabble/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1073 @@
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "string",
+                    "description": "id"
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "name"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get channels",
+        "operationId": "GetChannels",
+        "x-ms-visibility": "internal",
+        "parameters": [],
+        "description": "Get channels"
+      }
+    },
+    "/content/internalnews": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "An internal news",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "The unique identifier of the content",
+                  "title": "Id"
+                },
+                "contents": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "language": {
+                        "type": "string",
+                        "description": "Language",
+                        "title": "language"
+                      },
+                      "isDefault": {
+                        "type": "boolean",
+                        "description": "Is the language default",
+                        "title": "language default ?"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "Title",
+                        "title": "title"
+                      },
+                      "summary": {
+                        "type": "string",
+                        "description": "A short summary of the content",
+                        "title": "summary"
+                      },
+                      "description": {
+                        "type": "string",
+                        "description": "Content",
+                        "title": ""
+                      },
+                      "mediaIds": {
+                        "type": "array",
+                        "items": {
+                          "type": "string"
+                        },
+                        "description": "mediaIds"
+                      }
+                    }
+                  },
+                  "description": "",
+                  "title": "Content"
+                },
+                "channelIds": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "The channels that the content is published to",
+                    "title": "Channels"
+                  },
+                  "description": "channelIds"
```

Comment:
@mdujardibs Can it be `Channel Ids` ?

Reply:
Done

Reply:
@mdujardibs Can you please push your changes to the PR, the description values are not fixed